modules/op-release-notes-1-6.adoc

```diff
@@ -274,4 +273,30 @@ Error from server (InternalError): Internal error occurred: failed calling webho
 // https://issues.redhat.com/browse/SRVKP-1783
 
 * The pod serving the `tkn` CLI will now be scheduled on nodes, based on the node selector and toleration limits specified in the `TektonConfig` custom resource.
-// https://issues.redhat.com/browse/SRVKP-1804
\ No newline at end of file
+// https://issues.redhat.com/browse/SRVKP-1804
+
+
+[id="release-notes-1-6-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.2
+
+[id="fixed-issues-1-6-2_{context}"]
+=== Fixed issues
+
+* The Operator ensures that only one instance of each type of `TektonInstallerSet` exists after an upgrade.
+// https://issues.redhat.com/browse/SRVKP-1926
+
+* To decide resource recreation during an upgrade, the Operator uses the operator version instead of component version.
+// https://issues.redhat.com/browse/SRVKP-1928
+
+* After an upgrade, the Pipelines Webhook service persists in the cluster.
+// https://issues.redhat.com/browse/SRVKP-1939
+
+* Cron jobs for auto-prunning get recreated only if there is a relevant annotation change in the namespace.
+// https://issues.redhat.com/browse/SRVKP-1826
+
+* Upstream version of Tekton Pipelines has been revised to `v0.28.3`, which has the following fixes:
+** Fix pipeline/task to Run label/annotation propagation.
+** For implicit params:
+*** Do not apply the `PipelineSpec` params to TaskRefs.
+*** Disable implicit param behavior for `Pipeline` objects.
+// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
```

Comment:
Observation: I see this line on the GH page: [#4521](https://github.com/tektoncd/pipeline/pull/4521) Update Dockerfiles using golang images to Go 1.16.13

I believe that you have not mentioned it in the RN as it is not relevant. Please recheck and confirm. Thanks!

Reply:
Yes, it's not relevant based on SME Input.